src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -88,6 +88,7 @@ function BaseAnchorForCommentsOnly({onPressIn, onPressOut, href = '', rel = '',
                         event.preventDefault();
                         linkProps.onPress();
                     }}
+                    suppressHighlighting
```

Comment:
We shouldn't suppress here. Its already wrapped under `PressableWithSecondaryInteraction` which means its expecting to have opacity change on press. @c3024